profiles/openasu/CHANGELOG.txt

```diff
@@ -1,6 +1,48 @@
+Webspark 1.64 (California) release
+----------------------------------
+- Webspark core
+
+  * Drupal (v7.65)
+    * This release fixes security vulnerabilities. See https://www.drupal.org/project/drupal/releases/7.65 for more details.
+  * Panopoly (v1.64)
+    * This release includes a fix for a Moderately security vulnerability in Drupal core, as well as Panopoly contrib module updates.
```

Comment:
Typo. "Moderately" should be "Moderately Critical" I assume.

Reply:
It's been changed to 'Moderately Critical' Thanks Michael!